README.md

```diff
@@ -0,0 +1,50 @@
+# Community Base Addons
+
+[![](https://img.shields.io/badge/Changelog-2.0-orange.svg?style=flat-square)](https://github.com/CBATeam/CBA_A3/issues?q=milestone%3A%22CBA_A3+v2.0%22+is%3Aclosed)
+[![](https://img.shields.io/badge/Release-2.0-blue.svg?style=flat-square)](https://github.com/CBATeam/CBA_A3/releases/tag/v2.0.0.150817)
+[![](https://img.shields.io/badge/Github-Wiki-lightgrey.svg?style=flat-square)](https://github.com/CBATeam/CBA_A3/wiki)
+[![](https://img.shields.io/badge/Function-Documentation-yellow.svg?style=flat-square)](https://cbateam.github.io/CBA_A3/docs/files/overview-txt.html)
+
+## Installation
+
+Download the latest version and unpack in your ARMA installation folder.
+For ARMA 3 content, launch with `-mod=@CBA_A3`
+
+## Optionals
+
+PBO                                    | Description
+-------------------------------------- | --------------------------------------
+cba_cache_disable.pbo                  | Dev Tool. Copy this to your cba\addons if you want to disable CBA's caching of functions.
+cba_diagnostic_disable_xeh_logging.pbo | Copy this to your cba\addons if you want to disable all the extra RPT logging.
+cba_diagnostic_enable_logging.pbo      | Dev Tool. Copy this to your cba\addons if you want to enable more logging.
```

Comment:
To your `@CBA_A3/addons`...

Though this line is for all optionals, cleaner would be to put it before the table.